assets/sass/2_fragments/_forms.scss

```diff
@@ -876,7 +876,6 @@ fieldset,
 
     .form-field &,
     fieldset & {
-        // @include border-left(2px solid $color-light-beta);
```

Comment:
This only removes it from the Sass file. It would need to be compiled to css locally via Gulp.

That being said, I have already made this fix and [published to NPM](https://www.npmjs.com/~ushahidi-admin)